mm2src/coins/lightning/ln_errors.rs

```diff
@@ -225,15 +243,21 @@ impl From<CoinFindError> for ListChannelsError {
     }
 }
 
+impl From<SqlError> for ListChannelsError {
+    fn from(err: SqlError) -> ListChannelsError { ListChannelsError::SqlError(err.to_string()) }
+}
+
 #[derive(Debug, Deserialize, Display, Serialize, SerializeErrorType)]
 #[serde(tag = "error_type", content = "error_data")]
 pub enum GetChannelDetailsError {
     #[display(fmt = "Lightning network is not supported for {}", _0)]
     UnsupportedCoin(String),
     #[display(fmt = "No such coin {}", _0)]
     NoSuchCoin(String),
-    #[display(fmt = "Channel with id: {:?} is not found", _0)]
-    NoSuchChannel(H256Json),
+    #[display(fmt = "Channel with rpc id: {} is not found", _0)]
+    NoSuchChannel(u64),
+    #[display(fmt = "SQL error {}", _0)]
+    SqlError(String),
```

Comment:
This error will likely not happens in wasm, how about allowing it only in native?

Reply:
Right now LightningCoin is not supported in Wasm https://github.com/KomodoPlatform/atomicDEX-API/blob/6b2e98fda5775cd857e69a2438f80d6bc0a552f7/mm2src/coins/lp_coins.rs#L110 https://github.com/KomodoPlatform/atomicDEX-API/blob/6b2e98fda5775cd857e69a2438f80d6bc0a552f7/mm2src/coins/lp_coins.rs#L1461-L1462 I will take this into consideration when implementing Wasm, for now maybe I can rename this to DbError instead so it can include wasm DB errors in the future too.